homework2/sass/header.scss

```diff
@@ -0,0 +1,343 @@
+              transform: scale(0) translate(0px, -50%);
+            }
+
+            &::before {
+              top: 50%;
+              transform: rotate(-45deg) translate(0px -50%);
+              -webkit-transform: rotate(-45deg)
+            }
+
+            &::after {
+              top: 50%;
+              transform: rotate(45deg) translate(0px 50%);
+              -webkit-transform: rotate(45deg)
+            }
+          }
+        }
+
+        nav {
+          position: fixed;
+          top: 0;
+          left: -100%;
+          width: 100%;
+          height: 100%;
+          background-color: $color-primary;
+          padding-top: 92px;
+          transition: left 0.3s ease 0s;
+          overflow: auto;
+          z-index: 4;
+
+          ul {
+            li {
+              list-style-type: none;
+              display: flex;
+              flex-direction: column;
+              align-items: center;
+
+              a {
+                text-decoration: none;
+                text-transform: uppercase;
+                @include fontProp(12px, 24px, $color-link);
+                letter-spacing: 2px;
+                padding: 8px;
+
+                &:hover {
+                  @include fontProp(14px, 24px, $color-link-hover);
+                }
+
+                &.current {
+                  padding-bottom: 3px;
+                  display: flex;
+                  flex-direction: column;
+                  align-items: center;
+
+                  &:after {
+                    content: "";
+                    width: 40px;
+                    height: 2px;
+                    background-color: $color-link;
+                    display: block;
+                    text-align: center;
+                    transition: all 0.3s ease 0s;
+                  }
+                }
+              }
+
+
+              &.get-skout-btn, &.login-btn {
+                button {
+                  @include tpl-button;
+                  box-shadow: none;
+                  margin: 200px 0 15px;
+                  &:hover{
+                    box-shadow: none;
+                  }
+                }
+              }
+
+              &.login-btn {
+                button {
+                  padding: 8px 22px;
+                  margin: 0 0 40px;
+                }
+              }
+            }
+          }
+
+          &.active {
+            left: 0;
+
+            &::before {
+              content: "";
+              position: fixed;
+              width: 100%;
+              top: 0;
+              left: 0;
+              height: 62px;
+              background: linear-gradient(45deg, #12C1E6, #01F973);
+              z-index: 2;
+              border: none;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .header-container-fixed {
+    position: fixed;
+    background-color: $color-primary;
+
+    .menu-icon {
+      span, &::before, &::after {
+        background-color: $color-secondary-text !important;
```

Comment:
don't use !important, the only case when you can use it, whwn you need override a plugin styles

Reply:
Done